### PR TITLE
Correcting line length and other minor cleanup.

### PR DESCRIPTION
**Pull request by AnmAtAnm**
Also, adding fluent Subjects for View and Connection for test clarity (also shorter).

---
This change is [Reviewable](https://reviewable.io/reviews/google/blockly-android/479)

### CODE REVIEW

File: blocklytest/src/androidTest/java/com/google/blockly/utils/ConnectionSubject.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package com.google.blockly.utils;
```
Comment (picklesrus) [inline blocklytest/src/androidTest/java/com/google/blockly/utils/ConnectionSubject.java]: Does this file need the license?

File: blocklytest/src/androidTest/java/com/google/blockly/utils/ViewSubject.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package com.google.blockly.utils;
+
```
Comment (picklesrus) [inline blocklytest/src/androidTest/java/com/google/blockly/utils/ViewSubject.java]: same license question

File: blocklytest/src/androidTest/java/com/google/blockly/utils/ConnectionSubject.java (hunk 1 above)
Comment (AnmAtAnm) [inline blocklytest/src/androidTest/java/com/google/blockly/utils/ConnectionSubject.java]: Done.

File: blocklytest/src/androidTest/java/com/google/blockly/utils/ViewSubject.java (hunk 2 above)
Comment (AnmAtAnm) [inline blocklytest/src/androidTest/java/com/google/blockly/utils/ViewSubject.java]: Done.

**Pull request merged**